utility.py

```diff
@@ -83,24 +83,5 @@ def copy_current_platform_mssqltoolsservice():
         print("This platform: {} does not support mssqltoolsservice.".format(platform.system()))
 
 
-def encode(s):
```

Comment:
These were unused?

Reply:
Correct. We moved these methods to `mssqlcli/util.py` a few weeks back.